rclpy/rclpy/impl/rmw_implementation_tools.py

```diff
@@ -85,23 +83,8 @@ def import_rmw_implementation():
     if __rmw_implementation_module is not None:
         return __rmw_implementation_module
     available_implementations = get_rmw_implementations()
-    if __selected_rmw_implementation is None:
-        logger = logging.getLogger('rclpy')
-        # prefer FastRTPS, otherwise first in alphabetical order
-        # the same logic is implemented in
-        # rmw_implementation_cmake/cmake/get_default_rmw_implementation.cmake
-        default_impl = 'rmw_fastrtps_cpp'
-        if default_impl in available_implementations:
-            select_rmw_implementation(default_impl)
-        elif available_implementations:
-            select_rmw_implementation(available_implementations[0])  # select the first one
-        else:
-            raise NoImplementationAvailableException()
-        logger.debug("Implicitly selecting the '{0}' rmw implementation."
-                     .format(__selected_rmw_implementation))
-    module_name = '._rclpy__{rmw_implementation}'.format(
-        rmw_implementation=__selected_rmw_implementation,
-    )
+    __selected_rmw_implementation = 'rmw_implementation'
```

Comment:
Imo we have been building separate modules only for the reason that we were forced to do so. I don't think it was for having it as a feature. Also removing building N variations of the Python module is a benefit inmo.

I have removed the obsolete parts in 9e0597807ad9b64397ba35eaa549f569978a039e.

Anyway since you vote with `-0` I guess we have to wait for more votes on this then before moving forward. @ros2/team Please cast your vote here.